### PR TITLE
[EDIT] securing_nats tls.md re: TLS-Terminating Reverse Proxies

### PR DESCRIPTION
**Pull request by scottf**
Update the section regarding TLS-Terminating Reverse Proxies

### CODE REVIEW

File: running-a-nats-service/configuration/securing_nats/tls.md [hunk 1]
```diff
@@ -166,41 +166,22 @@ openssl x509 -noout -text -in client-cert.pem
...
-
-See: <https://github.com/nats-io/nats.rs/blob/main/async-nats/src/connector.rs>
-
-### nats.net
+Once this is configured, your client can connect to the proxy with normal (language specific) tls configuration. Please make sure you are using the appropriate version of your langauge specific client.
 
-See: <https://github.com/nats-io/nats.net/tree/main/src/Samples/TLSReverseProxyExample>
+| Client | Version |
+| --- | --- |
+| nats.go | ??? |
```
Comment (wallyqs) [inline running-a-nats-service/configuration/securing_nats/tls.md:182]: version that supports tls handshake first of the client should work

Comment (scottf) [reply running-a-nats-service/configuration/securing_nats/tls.md:182]: TLS Handshake is in https://github.com/nats-io/nats.go/releases/tag/v1.31.0

**Comment by gcolliso** [comment]
@scottf is this ready to merge or do we need further review?

**Comment by scottf** [comment]
@gcolliso It's ready

**Pull request merged**